vowpalwabbit/nn.cc

```diff
@@ -243,7 +230,7 @@ void predict_or_learn_multi(nn& n, single_learner& base, example& ec)
     float save_final_prediction = 0;
     float save_ec_loss = 0;
 
-CONVERSE:  // That's right, I'm using goto.  So sue me.
+  CONVERSE:  // That's right, I'm using goto.  So sue me.
```

Comment:
IIRC labels need to be at the beginning of the line

Reply:
This was clang-format being "helpful" will revert it